index.html

```diff
@@ -233,5 +233,41 @@ <h2 class="aboutMeTitle">
             </div>
         </div>
     </section>
+    <section>
+        <div class="contact">
+            <p class="newProjects">
+                I'm always interested in hearing about 
+                new projects, so if you'd like to chat 
+                please get in touch.
+            </p>
+            <form action="https://formspree.io/f/xoqzkakk" class="contactForm" method="post">
+                <input type="text" id="fname" name="fname" placeholder="Full Name" maxlength="30" required><br>
+                <input type="email" id="email" name="_replyto" placeholder="Enter email" required><br>
+                <textarea name="Opinion" id="Opinion" cols="50" rows="10" maxlength="500" placeholder="Write me something..." required></textarea>
+                <div class="btn-submitContainer">
+                    <button type="submit" id="submit" name="submit" value="submit">Get In Touch</button>
+                </div>
+            </form>       
+        </div>
+    </section>
+    <footer>
+        <img src="images/line3.png" alt="line for footer start">
+        <div>
+            <ul class="socialmedia2">
+                <li>
+                    <a href="https://github.com/akezeth"><img src="images/github2.png" alt="social media github"></a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/aklilu-mekonnen-a8287b74"><img src="images/linkedin2.png" alt="social media linkedin"></a>
+                </li>
+                <li><a href="#"><img src="images/angelist2.png" alt="social media angelist"></a></li>
+                <li><a href="https://twitter.com/akezeth"><img src="images/twitter2.png" alt="social media twitter"></a></li>
+                <li><a href="#"><img src="images/medium2.png" alt="social media mediam"></a></li>
+            </ul>
+        </div>
+        <div class="footer-bottomLine">
+           <img src="images/line4.png" alt="blank line image">
+        </div>
```

Comment:
- [ ] Kindly remove this element. In the design template on Figma, this line appears because it is a prototype that shows how it looks on Apple mobile phones. It isn't part of the design. I would advise that you remove it. 👍